### PR TITLE
chore: update class hash in genesis config

### PR DESCRIPTION
**Pull request by ayushtom**
# Pull Request type

Chore: added actual class hashes for contracts which had dummy values.

Please add the labels corresponding to the type of changes your PR introduces:

- Bugfix

## What is the current behavior?

Currently we add dummy values from certain contracts listed below - 
- UniversalDeployer.json
- test.json
- erc20.json
- erc721.json
Here the values were mocked. We add real values generated from starkli for the contracts and add these in the genesis config

Resolves: #1049 

## What is the new behavior?

- We add the class hashes for contracts with the help of starkli
- We add this from starkli since this also matches with the declared contract class hash on goerli

## Does this introduce a breaking change?

No

### CODE REVIEW

**Comment by ayushtom** [comment]
closes #1049

**Comment by ayushtom** [comment]
@tdelabro can you review these changes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Madara Changelog
 
+- chore: add real class hash values for genesis config
```
Comment (tdelabro) [inline CHANGELOG.md:3]: Goes under the `next release` section

**Review by tdelabro — CHANGES_REQUESTED** [review]
I thought those values also exit as hardcoded constants somewhere in the Rust code. Maybe I'm wrong, can you double check tho

File: CHANGELOG.md (hunk 1 above)
Comment (ayushtom) [reply CHANGELOG.md]: done

**Comment by ayushtom** [comment]
> I thought those values also exit as hardcoded constants somewhere in the Rust code. Maybe I'm wrong, can you double check tho

I think i see that rust code normally serialises this json and reads value from there. These values were not hardcoded in rust code implementation.

**Comment by ayushtom** [comment]
> I thought those values also exit as hardcoded constants somewhere in the Rust code. Maybe I'm wrong, can you double check tho

I think i see that rust code normally serialises this json and reads value from there. These values were not hardcoded in rust code implementation. @tdelabro

**Review by tdelabro — APPROVED** [review]

**Pull request merged**